css/styles.css

```diff
@@ -0,0 +1,65 @@
+
+body {background-color:#FFFFFF; 
+margin-left: 0px;
+margin-right: 0px;
+margin-top: 0px;
+margin-bottom: 0px;
+background="images/layout_39.gif" width:100% auto}
+
+table {border: 0px; border-collapse:collapse; width: 100%} 
+
+table td{padding:0px;}
+
+#t2, #t3, #t4 {border: 0px; border-collapse:collapse; bordercolor: #111111; width: 100%;}  
+
+#l1 {width: 364px; height: 28px}
```

Comment:
this is awful!!!

1) do not use ids. ids are used for javascript ONLY!!! use classes.
2) use class nesting, use global classes to avoid duplication and mess.
3) use descriptive class names so it is easily found and understandable.